test/teiserver/lobby/libs/lobby_restrictions_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule Teiserver.Lobby.Libs.LobbyRestrictionsTest do
+  @moduledoc false
+  use ExUnit.Case
+  alias Teiserver.Lobby.LobbyRestrictions
+
+  # test "raw call tests" do
+
+  # end
+
+  test "check for noob title" do
+    result = LobbyRestrictions.is_noob_title?("Noobs 1v1")
+    assert result == true
+
+    result = LobbyRestrictions.is_noob_title?("No Noobs 1v1")
+    assert result == false
+
+    result = LobbyRestrictions.is_noob_title?("All Welcome 1v1")
+    assert result == false
+
+    result = LobbyRestrictions.is_noob_title?("Newbies 1v1")
+    assert result == true
+
+    result = LobbyRestrictions.is_noob_title?("Nubs 1v1")
+    assert result == true
```

Comment:
For these boolean checks, probably a little bit cleaner and concise to do it like this:
```suggestion
    assert LobbyRestrictions.is_noob_title?("Noobs 1v1")
    
    refute LobbyRestrictions.is_noob_title?("No Noobs 1v1")
    
    refute LobbyRestrictions.is_noob_title?("All Welcome 1v1")
    
    assert LobbyRestrictions.is_noob_title?("Newbies 1v1")
    
    assert LobbyRestrictions.is_noob_title?("Nubs 1v1")
```